pom.xml

```diff
@@ -60,7 +60,10 @@
             <artifactId>derby</artifactId>
             <version>10.10.2.0</version>
         </dependency>
-
+        <dependency>
```

Comment:
I would add dependencies through its own PR, so after merge we can all rebase on it

Reply:
Removed.